roofit/roofitcore/inc/RooWorkspace.h

```diff
@@ -25,6 +25,10 @@
 #include <map>
 #include <list>
 #include <string>
+#include "ROOT/RMakeUnique.hxx"
+
+using std::unique_ptr;
+using std::make_unique;
```

Comment:
I think using statements like these are not allowed in headers.

Reply:
Oh, that makes a lot of sense ...
Good spot.